float32 x
float32 y
float32 slope
# image of object
uint16 imbufx
uint16 imbufy
sensor_msgs/Image image
